app/views/course/condition/_conditions.html.slim

```diff
@@ -0,0 +1,23 @@
+- conditional_name = conditional.class.model_name.human.downcase
```

Comment:
we typically do not use downcase for this, because the model name should be capitalised.